src/components/RouteUpdate.tsx

```diff
@@ -0,0 +1,300 @@
+                                    <Row align="top">
+                                        <Col flex="auto">
+
+                                        </Col>
+                                    </Row>
+
+                                </Header>
+                            </Col>
+                            <Col span={24}>
+
+                            </Col>
+                            <Col span={24}>
+                                <Form.Item
+                                    name="network"
+                                    label="Network CIDR"
+                                    tooltip="Use CIDR notation. e.g. 192.168.10.0/24 or 172.16.0.0/16"
+                                    rules={[{validator: networkRangeValidator}]}
+                                >
+                                    <Input placeholder="Add route network range..." ref={inputNameRef} onPressEnter={() => toggleEditName(false)} onBlur={() => toggleEditName(false)} autoComplete="off" minLength={9} maxLength={43}/>
+                                </Form.Item>
+                            </Col>
+                            <Col span={24}>
+                                <Form.Item
+                                    name="enabled"
+                                    label="Status"
+                                >
+                                    <Radio.Group
+                                        options={optionsDisabledEnabled}
+                                        optionType="button"
+                                        buttonStyle="solid"
+                                    />
+                                </Form.Item>
+                            </Col>
+
+                            <Col span={24}>
+                                <Form.Item
+                                    name="peer"
+                                    label="Routing peer"
+                                    tooltip="You can choose one routing peer or leave it empty"
+                                >
+                                    <Select
+                                            showSearch
+                                            style={{ width: '100%' }}
+                                            placeholder="Select Peer"
+                                            onChange={handlePeerChange}
+                                            dropdownRender={dropDownRender}
+                                            options={options}
+                                            allowClear={true}
+                                    />
+                                </Form.Item>
+                            </Col>
+                            <Col span={24}>
+                                <Form.Item
+                                    name="masquerade"
+                                    label="Masquerade"
+                                    tooltip="Enable masquerade or hiding the traffic with the routing peer address"
```

Comment:
The wording is a bit odd.
Maybe this?
`Enable masquerade or hide traffic with the routing peer address.`